reactor-core/src/main/java/reactor/core/publisher/DirectProcessor.java

```diff
@@ -81,7 +81,7 @@
  * @deprecated Prefer clear cut usage of either {@link Processors} or {@link Sinks}, to be removed in 3.5
  */
 @Deprecated
-public final class DirectProcessor<T> extends FluxIdentityProcessor<T> {
+public final class DirectProcessor<T> extends FluxProcessor<T, T> {
```

Comment:
could you maybe revert the `FluxIdentityProcessor` removal for now, and offer it in a separate PR ? this distracts from the rest of the PR IMHO, as maybe 90% of these files are only changed due to this removal.

Reply:
The PR is not finished and there will be more impact due to removing Processors and use of Processor directly wherever possible.